about.html

```diff
@@ -0,0 +1,126 @@
+            <div class="info-content">
+                <h2>Creative Commons 
+                    Global Summit 2015 Logo</h2>
+                    <img src="./images/Home/feature-title-underline.svg" alt="">
+                <p id="mobile">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+                <p id="desktop">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+                    After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+                    After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+            </div>
+            <div class="about_logo">
+                <img src="./images/About/about_logo.svg" alt="about_logo">
+            </div>
+        </section>
+        <section  class="info">
+            <div class="info-content">
+                <h2>See the past CC Global Summits</h2>
+                    <img src="./images/Home/feature-title-underline.svg"" alt="">
+                <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw.</p>
+            </div>
+            <div class="abt-img">
+                <img src="./images/About/past-summit-image1.svg" alt="past-summit-image1">
+                <img src="./images/About/past-summit-image2.svg" alt="past-summit-image2">
+            </div>
+        </section>
+
+        <section class="partner" id="mobile">
+            <div class="partner-content">
+                <h2>Partners</h2>
+                <img src="./images/Home/feature-title-underline.svg" alt="">
+                <div class="part-logo">
+                    <ul>
+                        <li><img src="./images/About/mozilla-logo.svg" alt="mozilla-logo"></li>
+                        <li><img src="./images/About/googlekorea-logo.svg" alt="googlekorea-logo"></li>
+                        <li><img src="./images/About/naver-logo.svg" alt="naver-logo"></li>
+                        <li><img src="./images/About/daumkakao-logo.svg" alt="daumkakao-logo"></li>
+                        <li><img src="./images/About/airbnb-logo.svg" alt="airbnb-logo"></li>
+                    </ul>
+                </div>
+            </div>   
+        </section>
+        <footer class="footer">
+            <div class="footer-logo" id="mobile"> <img src="./images/About/footer-logo.svg" alt="footer-logo"></div>
+            <div class="footer-logo" id="desktop"> <img src="./images/About/cck-logo-about.svg" alt="footer-logo"></div>
+            <div class="footer-des" ><p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+            <p id="desktop">달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+                자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요.</p></div>
+        </footer>
```

Comment:
- [ ] Hi, I notice that the footer of the About page is surrounded with white color. Please work on it again and follow the design as stated in your project requirements,
- **Your design:**
![Capture](https://user-images.githubusercontent.com/63972670/235298314-8a397a44-9c19-4314-843a-0f525a818d66.JPG)
- Original Design:
![Capture](https://user-images.githubusercontent.com/63972670/235298359-ae2d9342-276b-49ab-ae14-6112841e1879.JPG)